src/snowflake/snowpark/dataframe_reader.py

```diff
@@ -1116,6 +1116,10 @@ def dbapi(
         custom_schema: Optional[Union[str, StructType]] = None,
         predicates: Optional[List[str]] = None,
         session_init_statement: Optional[Union[str, List[str]]] = None,
+        imports: Optional[List[str]] = None,
+        packages: Optional[List[str]] = None,
+        external_access_integration: Optional[str] = None,
+        ingestion_mode: Literal["cloud_mode", "local_mode"] = "parquet_mode",
```

Comment:
you will need to change "udtf_mode" and "parquet_mode" in your code, but let's get Yijun's feedback first

Reply:
got it, lets wait for Yijun' comment then